src/Controls/tests/TestCases.HostApp/Issues/Issue29595_29634.cs

```diff
@@ -0,0 +1,366 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace Maui.Controls.Sample.Issues
```

Comment:
This test is really complex and there are many things here. Does this issue not repro in a case without all of this special work?

I see multiple collection views, shell, pages, tabs and all sorts of delay things.

Can this not be broken into a simple test where the page loads and then the UI test taps a button to add an item if we need things loaded later.

Also, the fix for the empty view has no tests I think?